doc/nrf/includes/build_and_run_nrf9160.txt

```diff
@@ -1,7 +1,7 @@
 This sample can be found under |sample path| in the |NCS| folder structure.
 
 The sample is built as a non-secure firmware image for the ``nrf9160dk_nrf9160_ns`` build target.
-Because of this, it automatically includes the :ref:`secure_partition_manager`.
-You can also configure it to use :ref:`TF-M <ug_tfm>` instead of SPM.
+Because of this, it automatically includes the :ref:`Trusted Firmware-M <ug_tfm>` (TF-M).
+You can also configure it to use the :ref:`secure_partition_manager` instead of TF-M.
```

Comment:
```suggestion
You can configure it to use the :ref:`secure_partition_manager` instead of TF-M.
```
I don't think that's an "also" situation, though you could replace it with "optionally".